configure.ac

```diff
@@ -821,7 +821,7 @@ fi
 AC_MSG_CHECKING([whether to build fuzzers])
 if test "$enable_fuzzers" = "yes"; then
     AC_MSG_RESULT([yes])
-    LIBFUZZER_FLAGS="-fsanitize=fuzzer-no-link"
+    LIBFUZZER_FLAGS="-fsanitize=fuzzer-no-link,address"
```

Comment:
Same here. Jenkins enables both ASAN and UBSAN for fuzzers, btw.

Reply:
Perhaps we should enable more sanitizers here too?

Reply:
It's basically your sandbox currently, so up to you. :-) If you want to get closer to the Jenkins setup, then enabling both "address" and "undefined" probably makes sense.

Reply:
I've opted to leave this up to the user. I do remind in the help that configuring `--with-sanitizer=address,undefined` is a good idea, but don't force it. This maximizes flexibility.